core/templates/dev/head/pages/exploration_editor/translation_tab/welcome_translation_modal_directive.html

```diff
@@ -0,0 +1,92 @@
+<div class="modal-body oppia-long-text protractor-test-welcome-modal">
+  <div class="row">
+    <div class="col-lg-8 col-md-8 col-sm-8 col-xs-7">
+      <h1 class="oppia-welcome-modal-h1">Welcome!</h1>
+      <p class="oppia-welcome-modal-p">
+        It looks like you're new to translating explorations.
+      </p>
+      <p class="oppia-welcome-modal-p">
+        Would you like an introduction to translations in Oppia?
+      </p>
+
+      <button type="button"
+              class="btn oppia-welcome-modal-button oppia-transition-200 protractor-test-start-tutorial"
+              ng-click="beginTutorial()">
+        Yes, please!
+      </button>
+
+      <button type="button"
+              class="btn oppia-welcome-modal-button oppia-transition-200 protractor-test-dismiss-welcome-modal"
+              ng-click="cancel()">
+        Nope, I'm good.
+      </button>
+    </div>
+
+    <div class="col-lg-4 col-md-4 col-sm-4 col-xs-5">
+      <img class="oppia-welcome-modal-img" ng-src="<[editorWelcomeImgUrl]>">
+    </div>
+  </div>
+</div>
+
+<style>
```

Comment:
If these styles are similar to editor welcome modal the can we move this to oppia.css?

@seanlip, will this be fine or do we want to have separate CSS for each template?

Reply:
They should be very similar.